src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //равнобедренная трапеция
+    private double height; // высота
+
+    public IsoscelesTrapezoid(String color, double firsSide, double secondSide, double height) {
+        super(color, firsSide, secondSide);
+        this.height = height;
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return ((getFirsSide() + getSecondSide()) / 2) * height;
+    }
+
+    @Override
+    public String infoFigure() {
+        return ("Длина = " + getFirsSide() + " см ,"
+                + " Ширина = " + getSecondSide() + " см , "
+                + "Высота = " + height + " см , "
+                + " Площадь трапеция = " + detAreaOfFigure()) + " см";
```

Comment:
Avoid using non-English text in your code. The checklist advises using only English in messages and code to maintain universality and consistency.